scripts/generate_proto_stubs.sh

```diff
@@ -73,7 +73,7 @@ PROTO_FILES=$(grep "GenProto.*google" $PYTHON_PROTOBUF_DIR/python/setup.py | \
 # shellcheck disable=SC2086
 protoc_install/bin/protoc --proto_path="$PYTHON_PROTOBUF_DIR/src" --mypy_out="relax_strict_optional_primitives:$REPO_ROOT/stubs/protobuf" $PROTO_FILES
 
-isort "$REPO_ROOT/stubs/protobuf"
+ruff "$REPO_ROOT/stubs/protobuf" --exit-non-zero-on-fix --fix-only
```

Comment:
I think `--exit-non-zero-on-fix` is backwards. If fixing happens, we want ruff to exit with success so that the script continues. It uses `set -e`, so nonzero exit code would stop the script here.

Reply:
oops, you're right.
I'm also wondering whether `--fix-only` is wanted here. If you're gonna get errors later, may as well report them now? As long as it doesn't block script execution.